CODING.md

```diff
@@ -0,0 +1,59 @@
+# Coding guide
+
+This project is written in Go programming language and follows a community accepted coding styles and practices defined in https://golang.org/doc/effective_go.html and https://github.com/golang/go/wiki/CodeReviewComments. This document defines additional guidelines specific to the Bee project.
```

Comment:
since this is markdown, why not add text to the hyperlinks? like so:

`[Effective Go](https://golang.org/doc/effective_go.html)`